addons/udes_sale_stock/tests/test_sale_order.py

```diff
@@ -0,0 +1,108 @@
+from odoo.addons.udes_stock.tests import common
+from odoo import fields
+
+
+class TestSaleOrder(common.BaseUDES):
+    @classmethod
+    def setUpClass(cls):
+        super(TestSaleOrder, cls).setUpClass()
+        # Unlink the TestGoodsOut pull_push rule as it was causing the route to misbehave
```

Comment:
It would be better to explain precisely how the route misbehaved.  That way we know what to look out for if we reinstate the rule, and can distinguish any additional unexpected behaviour.